src/main/java/io/fabric8/elasticsearch/plugin/OpenshiftAPIService.java

```diff
@@ -65,6 +65,7 @@ public String userName(final String token) {
         try (DefaultOpenShiftClient client = factory.buildClient(token)) {
```

Comment:
`factory.buildClient(token)` - what does `buildClient` do with the `token`?

Reply:
It was my understanding that it added the to the header, but we are using the raw client this one wrappers so not sure about what it does.  The reality is we could use a straight REST client without kubernetes-client as we don't use any of the features it provides other then maybe the CA look up, but Its likely not worth the effort to remove it.